lib/v2/mtime/news.js

```diff
@@ -0,0 +1,50 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const link = 'http://news.mtime.com/';
+    const response = await got({
+        method: 'get',
+        url: link,
+    });
+    const $ = cheerio.load(response.data);
+    const list = $('ul.left-cont.fix li')
+        .slice(0, 10)
+        .map((_, item) => {
+            item = $(item);
+            return {
+                title: item.find('h4 a').text(),
+                link: item.find('h4 a').attr('href'),
+            };
+        })
+        .get();
+
+    const items = await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const detailResponse = await got({
+                    method: 'get',
+                    url: item.link,
+                });
+                const content = cheerio.load(detailResponse.data);
+
+                content('.DRE-subject-wrapper').remove();
+
+                content('.left .ps').remove();

```

Comment:
`.body` does not include `.left .ps`, so you can safely remove this.
Also, you can add `item.author` since the article provides it.
```suggestion
                item.author = content('.editor').text().replace('编辑：', '');
```

Reply:
Thank you.